File: csharp/Balance.cs
// Example showing how to get account balance
using System;
using System.Threading.Tasks;
using BinaryOptionsToolsUni;

class BalanceExample
{
    static async Task Main(string[] args)
    {
        try
        {
            // Initialize client
            var client = await PocketOption.NewAsync("your-session-id");
            
            // IMPORTANT: Wait for connection to establish
            await Task.Delay(5000);
            
            // Get current balance
            var balance = await client.BalanceAsync();
            Console.WriteLine($"Your current balance is: ${balance:F2}");
        }
        catch (Exception ex)
        {
            Console.WriteLine($"Error: {ex.Message}");
        }
    }
}

File: kotlin/Balance.kt
// Get balance example
import binary_options_tools_uni.*
import kotlinx.coroutines.*

suspend fun main() {
    val client = PocketOption.new("your-session-id")
    delay(5000)
    
    val balance = client.balance()
    println("Your current balance is: $$balance")
}
